worker/test/src/RTC/TestRtpStreamSend.cpp

```diff
@@ -26,85 +51,58 @@ SCENARIO("NACK and RTP packets retransmission", "[rtp][rtcp]")
-			0b01011011, 0b01101011, 0b11001010, 0b10110101,
-			0, 0, 0, 2
-		};
-		// clang-format on
+		0b10000000, 0b01111011, 0b01010010, 0b00001110,
+		0b01011011, 0b01101011, 0b11001010, 0b10110101,
+		0, 0, 0, 2
+	};
+	// clang-format on
 
-		uint8_t rtpBuffer2[65536];
-		uint8_t rtpBuffer3[65536];
-		uint8_t rtpBuffer4[65536];
-		uint8_t rtpBuffer5[65536];
+	uint8_t rtpBuffer2[1500];
+	uint8_t rtpBuffer3[1500];
+	uint8_t rtpBuffer4[1500];
+	uint8_t rtpBuffer5[1500];
 
-		// packet1 [pt:123, seq:21006, timestamp:1533790901]
-		RtpPacket* packet1 = RtpPacket::Parse(rtpBuffer1, sizeof(rtpBuffer1));
-
-		REQUIRE(packet1);
-		REQUIRE(packet1->GetSequenceNumber() == 21006);
-		REQUIRE(packet1->GetTimestamp() == 1533790901);
+	std::memcpy(rtpBuffer2, rtpBuffer1, sizeof(rtpBuffer1));
+	std::memcpy(rtpBuffer3, rtpBuffer1, sizeof(rtpBuffer1));
+	std::memcpy(rtpBuffer4, rtpBuffer1, sizeof(rtpBuffer1));
+	std::memcpy(rtpBuffer5, rtpBuffer1, sizeof(rtpBuffer1));
 
+	SECTION("receive NACK and get retransmitted packets")
+	{
+		// packet1 [pt:123, seq:21006, timestamp:1533790901]
+		auto packet1 = CreateRtpPacket(rtpBuffer1, 21006, 1533790901);
 		// packet2 [pt:123, seq:21007, timestamp:1533790901]
-		RtpPacket* packet2 = packet1->Clone(rtpBuffer2);
-
-		packet2->SetSequenceNumber(21007);
-		packet2->SetTimestamp(1533790901);
-
-		REQUIRE(packet2->GetSequenceNumber() == 21007);
-		REQUIRE(packet2->GetTimestamp() == 1533790901);
-
+		auto packet2 = CreateRtpPacket(rtpBuffer2, 21007, 1533790901);
 		// packet3 [pt:123, seq:21008, timestamp:1533793871]
-		RtpPacket* packet3 = packet1->Clone(rtpBuffer3);
-
-		packet3->SetSequenceNumber(21008);
-		packet3->SetTimestamp(1533793871);
-
-		REQUIRE(packet3->GetSequenceNumber() == 21008);
-		REQUIRE(packet3->GetTimestamp() == 1533793871);
-
+		auto packet3 = CreateRtpPacket(rtpBuffer3, 21008, 1533793871);
 		// packet4 [pt:123, seq:21009, timestamp:1533793871]
-		RtpPacket* packet4 = packet1->Clone(rtpBuffer4);
-
-		packet4->SetSequenceNumber(21009);
-		packet4->SetTimestamp(1533793871);
-
-		REQUIRE(packet4->GetSequenceNumber() == 21009);
-		REQUIRE(packet4->GetTimestamp() == 1533793871);
-
+		auto packet4 = CreateRtpPacket(rtpBuffer4, 21009, 1533793871);
 		// packet5 [pt:123, seq:21010, timestamp:1533796931]
-		RtpPacket* packet5 = packet1->Clone(rtpBuffer5);
+		auto packet5 = CreateRtpPacket(rtpBuffer5, 21010, 1533796931);
 
-		packet5->SetSequenceNumber(21010);
-		packet5->SetTimestamp(1533796931);
-
-		REQUIRE(packet5->GetSequenceNumber() == 21010);
-		REQUIRE(packet5->GetTimestamp() == 1533796931);
+		// Create a RtpStreamSend instance.
+		TestRtpStreamListener testRtpStreamListener;
 
 		RtpStream::Params params;
 
-		params.ssrc      = packet1->GetSsrc();
+		params.ssrc      = 1111;
 		params.clockRate = 90000;
 		params.useNack   = true;
 
-		// Create a RtpStreamSend.
-		RtpStreamSend* stream = new RtpStreamSend(&testRtpStreamListener, params, 4);
+		std::string mid{ "" };
```

Comment:
We don't need to initialize a string to "", it's that by default plus we are not consistent here (in other files we don't initialize it this way).